wolvic/wolvic_content_browser_client.cc

```diff
@@ -18,6 +19,18 @@ namespace {
 
 WolvicContentBrowserClient* g_instance = nullptr;
 
+// Adds the given platform to the user agent string. For example, if
+// we add "Mobile VR" platform to the following user agent
+// "Mozilla/5.0 (Linux; Android 8.0.0; Quest 2)", the result will be:
+// "Mozilla/5.0 (Linux; Android 8.0.0; Quest 2; Mobile VR)".
+void AddPlatformToUserAgent(const std::string& platform,
+                            std::string* user_agent) {
```

Comment:
Another nitpick. If we don't expect to ever call AddPlatformToUserAgent with a `user_agent` that is a `nullptr`, then better use a non-const reference:

```
void AddPlatformToUserAgent(const std::string& platform, std::string& user_agent)
```